nwb_conversion_tools/gui/nwb_conversion_gui.py

```diff
@@ -28,9 +28,9 @@
 
 
 class Application(QMainWindow):
-    def __init__(self, metafile=None, conversion_module='', source_paths={},
-                 kwargs_fields={}, extension_modules={}, extension_forms={},
-                 show_add_del=False):
+    def __init__(self, metafile=None, conversion_module=None, source_paths=None,
+                 kwargs_fields=None, extension_modules=None, extension_forms=None,
+                 show_add_del=False, nwbfile_loc=None, conversion_class=None):
         super().__init__()
```

Comment:
Can you add a Numpy-style docstring here? If a `conversion_module` and a `conversion_class` are both entered, you should throw an error.